File: Tournonment/AsemaneShekarAbadS1.java
import java.math.BigDecimal;
import java.math.RoundingMode;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {

            int row = scanner.nextInt();
            int column = scanner.nextInt();

            String[][] data = new String[row][column];
            int count = 0;

            for(int i = 0; i < row; i++) {
                String input = scanner.next();
                for(int j = 0; j < column; j++) {
                    data[i][j] = String.valueOf(input.charAt(j));
                    if (data[i][j].equals("*"))
                        count++;
                }
            }
            System.out.print(count);
        }
    }
}

// </Ctrl+Z>

File: Tournonment/AsemaneShekarAbadS1.py
import sys

def main():
    row, column = map(int, input().split())
    data = [['' for _ in range(column)] for _ in range(row)]
    count = 0

    for i in range(row):
        input_str = input()
        for j in range(column):
            data[i][j] = input_str[j]
            if data[i][j] == '*':
                count += 1

    print(count)

if __name__ == "__main__":
    main()
